go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/terraform-json v0.13.0 // indirect
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.7.0
-	github.com/jfrog/jfrog-client-go v0.27.0
+	github.com/jfrog/jfrog-client-go v1.10.0
```

Comment:
@alexhung - this needs to go.

Reply:
Agree. I'll create a ticket and schedule it soon.

Reply:
@Guent4 Some context with regards to `jfrog-client-go`. We have been moving away from using `jfrog-client-go` for a while now and use Resty client instead. Replication plus couple of other resources are still using structs from `jfrog-client-go`. Our plan is to replace them with local structs and remove all dependency of that package.